### ISSUE TITLE
allow multiple arguments for some commands

### ISSUE THREAD

**Issue by brrzap** [magicalraccoon/tootstream#96]
commands like `delete`, `boost`, `reject`, etc could allow more than one of whatever argument they take.  this would probably be used most by someone wanting to boost or fav a few posts at a time, or by someone working through their follow requests list.  of the current commands, these seem eligible:

* `delete`
* `boost` / `unboost`
* `fav` / `unfav`
* `block` / `unblock`
* `follow` / `unfollow`
* `mute` / `unmute`
* `accept` / `reject`
* `search` (?)

**Comment by magicalraccoon** [comment]
Does the api allow multiple action at once like that?

**Comment by brrzap** [comment]
no, it's generally not supported in a single call, at least by Mastodon.py.

we'd loop through the user's input and do an api call for each one.  this is fine for clients to do; the server applies a rate limit to prevent any client from taking up too much of its time.  i don't think we'll run into trouble when deleting a handful of toots, but if needed we could add a brief `sleep()` between each api call.

some api calls will accept a list (eg `Mastodon.account_relationships()`, which tootstream doesn't access right now), but it's uncommon in current Mastodon.py.
